core/templates/dev/head/galleries/Gallery.js

```diff
@@ -218,7 +218,7 @@ oppia.controller('Gallery', [
     });
 
     // Below is the width of each tile.
-    var TILE_DISPLAY_WIDTH = 211.953;
+    var TILE_DISPLAY_WIDTH = 208;
```

Comment:
made it so that it links with the css with 1 DOM request. there is an initialization to 208px in case things go wrong.